tests/network/test_services.py

```diff
@@ -1,7 +1,10 @@
-import bitcash
 import os
-import pytest
 import time
+import copy
+import unittest
```

Comment:
Not a particular fan of using unittest and pytests together.

Here's how we can setup and teardown in pytest: https://shay-palachy.medium.com/temp-environment-variables-for-pytest-7253230bd777

Reply:
Thanks I'll have a look.

Reply:
Done!

Note that unittest and pytest is already being used together 😅
For this reason I removed `@decorate_methods(catch_errors_raise_warnings, NetworkAPI.IGNORED_ERRORS)`. I don't think we should be ignoring those errors anyway, as they can alert us when providers are down (which happens a lot!!)

Reply:
On that note, some random 500 from Actorforth:

<img width="1112" alt="Screenshot 2024-04-27 at 17 13 32" src="https://github.com/pybitcash/bitcash/assets/21960643/903df408-269c-4daf-85d3-848dd8d61f89">

Not in the scope of this PR though.